html/php-challenge/favorite.php

```diff
@@ -0,0 +1,25 @@
+<?php
+session_start();
+require('dbconnect.php');
+
+if (isset($_SESSION['id'])) {
+  $id = $_REQUEST['id'];
+
+  $favs = $db->prepare('SELECT COUNT(*) as cnt FROM favorites WHERE member_id=? AND post_id=?');
+  $favs->execute(array($_SESSION['id'], $id));
+  $fav = $favs->fetch();
+
+  if ($fav['cnt'] == 0) {
```

Comment:
```suggestion
  if ($fav['cnt'] === 0) {
```
これは教科書がよくないのですが、厳密な比較演算子 `===`, `!==` を使うようにしましょう。 
教科書のもともとのコードは修正しなくてよいので、今回自分の書いたコードは曖昧な比較演算子を全て厳密な比較演算子に修正してください。